_maps/RandomRooms/backstreets/bossroom/slime_office.dmm

```diff
@@ -0,0 +1,2209 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"ai" = (
+/obj/effect/turf_decal/siding/green{
+	color = "#006400";
+	dir = 10
+	},
+/obj/effect/decal/cleanable/dirt/dust,
+/mob/living/simple_animal/hostile/ordeal/steel_dawn,
+/turf/open/floor/carpet/green,
+/area/city/backstreets_room)
+"ao" = (
+/obj/machinery/smartfridge/extract/preloaded,
+/turf/open/floor/plating{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"at" = (
+/turf/open/floor/plating/asteroid/snow/airless,
+/area/city/backstreets_room)
+"aY" = (
+/obj/effect/turf_decal/siding/purple{
+	dir = 1
+	},
+/turf/open/floor/plasteel/shuttle{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"bk" = (
+/obj/effect/turf_decal/siding/green{
+	color = "#006400";
+	dir = 4
+	},
+/obj/effect/decal/cleanable/dirt/dust,
+/turf/open/floor/plasteel/shuttle{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"bl" = (
+/obj/effect/turf_decal/siding/green{
+	color = "#006400";
+	dir = 4
+	},
+/turf/open/floor/plating{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"bw" = (
+/obj/effect/turf_decal/siding/purple,
+/obj/item/trash/sosjerky,
+/turf/open/floor/plasteel/shuttle{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"bA" = (
+/obj/effect/turf_decal/siding/purple{
+	dir = 1
+	},
+/obj/structure/table/wood/fancy/green,
+/obj/structure/displaycase/forsale,
+/obj/effect/decal/cleanable/cobweb/cobweb2,
+/turf/open/floor/plating{
+	color = "#42f545"
+	},
+/area/city/backstreets_room)
+"bJ" = (
+/obj/item/slimepotion/slime/docility,
+/obj/item/slimepotion/slime/docility,
+/obj/item/slimepotion/slime/docility,
+/turf/open/floor/plasteel/white,
+/area/city/backstreets_room)
+"ci" = (
+/obj/structure/chair/office,
+/obj/machinery/light{
+	dir = 1;
+	color = #32a852
```

Comment:
Me when this isn't a string (Mapdiffbot2's output is helpfull)
```suggestion
	color = "#32a852"
```

Reply:
Thanks, I just noticed that I missed a few of those and tried to fix them in the Fixing Slime Commit. But looks like I still missed a few.